terraform/datadog/variables.tf

```diff
@@ -7,3 +7,8 @@ variable "job_instances" {
   description = "List of pairs <job_name>:<job_count> of expected bosh job instance count"
   default     = []
 }
+
+variable "support_email" {
+  description = "DeskPro email address"
+  default     = "govpaas-alerting-dev@digital.cabinet-office.gov.uk"
```

Comment:
you can omit default. this will require the address to be always defined explicitly. which is sometimes a good form of check...